Casks/font/font-n/font-new-computer-modern.rb

```diff
@@ -0,0 +1,56 @@
+cask "font-new-computer-modern" do
+  version "6.0.0"
+  sha256 "1013f36abab243e74ad892c443358896a6bdbc3e295286873e95c04cf8fe13c4"
+
+  url "https://download.gnu.org.ua/release/newcm/newcm-#{version}.txz"
+  name "New Computer Modern"
+  homepage "https://download.gnu.org.ua/release/newcm/"
```

Comment:
```suggestion
  homepage "https://git.gnu.org.ua/newcm.git/about/"
```

It seems like the font has no dedicated website, but the repository might still provide more information.